templates/partners/desktop.html

```diff
@@ -6,144 +6,243 @@
-        <div class="p-card__content">
-          <img src="https://assets.ubuntu.com/v1/f627fbfb-hp.svg" alt="HP" width="80" height="80">
+        <div class="col-3">
+          <div class="long-white-rectangle">
+            <img src="https://assets.ubuntu.com/v1/f627fbfb-hp.svg" alt="HP" width="80" height="80">
+          </div>
           <p>HP sells a wide range of Canonical-certified Ubuntu laptops and desktop machines globally.</p>
+          <hr>
+          <a href="https://certification.ubuntu.com/desktop/models?vendors=HP"><span>View HP with Ubuntu</span></a>
         </div>
-        <a href="https://certification.ubuntu.com/desktop/models?vendors=HP" class="p-button--positive p-card__cta"><span>View HP with Ubuntu</span></a>
-      </div>
-      <div class="col-4 p-card--highlighted has-cta">
-        <div class="p-card__content">
-          <div class="u-vertically-center" style="height: 80px;">
+        <div class="col-3">
+          <div class="long-white-rectangle">
             <img src="https://assets.ubuntu.com/v1/3180a6b5-partner-logo-lenovo.svg" alt="Lenovo" width="195" height="40">
           </div>
           <p>Canonical has been shipping Ubuntu on a variety of Lenovo's popular machines which are available from Lenovo's enterprise teams.</p>
+          <hr>
           <p><a href="https://www.lenovo.com/us/en/think-workstations/artificial-intelligence-industry/c/artificial-intelligence-industry">Buy AI workstations with Ubuntu</a></p>
+          <hr>
+          <a href="https://certification.ubuntu.com/desktop/models?vendors=Lenovo"><span>View Lenovo with Ubuntu</span></a>
         </div>
-        <a href="https://certification.ubuntu.com/desktop/models?vendors=Lenovo" class="p-button--positive p-card__cta"><span>View Lenovo with Ubuntu</span></a>
       </div>
     </div>
-    <div class="row">
-      <div class="p-card--highlighted has-cta">
-        <div class="row">
-          <div class="col-8 u-vertically-center">
-            <h2 class="p-heading--4">Buy Ubuntu Advantage</h2>
-            <p style="max-width: none;">Security, compliance and support to improve efficiency while reducing complexity and cost.</p>
-          </div>
-          <div class="col-4 u-vertically-center" style="justify-content: inherit;">
-            <a href="https://buy.ubuntu.com/" class="p-button--positive"><span>Learn about our support options</span></a>
-          </div>
-        </div>
-      </div>
+  </div>
+</section>
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule">
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Get Ubuntu Pro</h2>
     </div>
-  </section>
+    <div class="col">
+      <p>Security, compliance and support to improve efficiency while reducing complexity and cost.</p>
+      <hr>
+      <p><a href="https://buy.ubuntu.com/">Learn about our support options</a></p>
+    </div>
+  </div>
+</section>
+
+<style>
+  .long-white-rectangle{
+    height: 420px;
```

Comment:
```suggestion
    aspect-ratio: .67
```
should look like so:
![image](https://github.com/canonical/canonical.com/assets/2741678/d9ac806f-96aa-4e3e-a92f-7e799fe3dc46)